bigbluebutton-html5/client/main.jsx

```diff
@@ -37,6 +37,9 @@ function loadLib(libname, success, fail) {
 };
 
 Meteor.startup(() => {
+
+  localStorage.clear();
```

Comment:
Then why are the font being saved on the localStorage? The point of using it is to not change the values when the user reload the page.